sortedm2m_tests/models.py

```diff
@@ -1,22 +1,18 @@
 # -*- coding: utf-8 -*-
 from django.db import models
-from django.utils.encoding import python_2_unicode_compatible
+from django.utils.encoding import force_text, python_2_unicode_compatible
```

Comment:
The build fails because `python_2_unicode_compatible` doesn't exist anymore in Django 2.2. We should provide a fallback if this import fails in `compat.py`. In case of `ImportError`, we can assume that we run Python 3, so the fallback can be:

```python
def python_2_unicode_compatible(klass):
    return klass
```